README.md

```diff
@@ -19,3 +19,146 @@ files. CMake developers document their functions by immediately proceeding the
 function with a comment block written in reStructuredText (reST). CMakeDoc will
 extract the comment blocks verbatim and create `*.rst` files which can be
 included in a normal Sphinx documentation set-up.
+
+## Installing
```

Comment:
I'm a strong believer that the README should just give you a project's status (the buttons), a quick description, and redirect you to ReadTheDocs for anything beyond that (like installing and detailed usage). IMHO, putting the documentation in the README overwhelms potential users when they navigate to the repo. It also tends to be harder to navigate a README than ReadTheDocs.

Reply:
Yeah that makes sense, I'll remove it after I migrate it over to the RST docs.